interface/src/app/gene/network/network.js

```diff
@@ -235,16 +255,29 @@ angular.module('adage.gene.network', [
           .call(edgeTip);
 
         geneTip.html(getGeneInfo);
-        network.onGene('mouseover.custom', geneTip.show);
-        network.onGene('mouseout.custom', geneTip.hide);
-        network.onEdge('mouseover.custom', showEdgeTip);
-        network.onEdge('mouseout.custom', hideEdgeTip);
+        network.onGene('click.custom', geneTip.show);
+        network.onEdge('click.custom', showEdgeTip);
```

Comment:
I agree that discovery may be tough. Can we create an issue to improve the discoverability of this feature? I agree that we should do it. I think that we'll have to get actual users touching it to find out if it's a problem. The issue will help us remember to do this.